### PR TITLE
decentralize environments

### LINKED ISSUE #77: Trouble installing environment
I'm having trouble installing the environment.

> $ conda env create -f environment.yml
> Warning: you have pip-installed dependencies in your environment file, but you do not list pip itself as one of your conda dependencies.  Conda may not use the correct pip to install your packages, and they may end up in the wrong place.  Please add an explicit pip dependency.  I'm adding one for you, but still nagging you.

There were some more pip errors, and then when I tried to activate the environment, I got: 

> Could not find conda environment: stat-rethink-pymc3
> You can list all discoverable environments with `conda info --envs`.

Should I try to install PyMC3 in my deafult environment instead?

### PR DESCRIPTION
**Pull request by aloctavodia**
Fix #77 and #68

### CODE REVIEW

File: Rethinking_2/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+## Display notebooks
+[![Binder](https://mybinder.org/badge.svg)](https://mybinder.org/v2/gh/pymc-devs/resources/master?filepath=Rethinking_2)
+[<img src="http://nbviewer.jupyter.org/static/img/nav_logo.svg" width=120>](http://nbviewer.jupyter.org/github/pymc-devs/resources/blob/master/Rethinking_2)
+
+
+## Contributing
+
+All contributions are welcome!
+
+Feel free to send PR to fix errors, improve the code, or make comments that could help users of this repository and readers of the book.
```
Comment (ColCarroll) [inline Rethinking_2/README.md:16]:
```suggestion
Feel free to send a PR to fix errors, improve the code, or make comments that could help users of this repository and readers of the book.
```

File: BCM/environment.yml [hunk 2]
```diff
@@ -6,5 +6,6 @@ dependencies:
   - mkl-service
   - seaborn
   - statsmodels
+  - pip
```
[the remark this answers is not in the document]
Comment (canyon289) [reply BCM/environment.yml:9]: I believe so

File: Rethinking_2/README.md (hunk 1 above)
Comment (canyon289) [reply Rethinking_2/README.md:16]: Fixed. Thanks

**Comment by canyon289** [comment]
LGTM

**Pull request merged**